lib/tcpip/tcpip_error_handler.h

```diff
@@ -12,7 +12,7 @@
 #include <simulator.h>
 #include <vector>
 
-using DebugErrorHandler = ConditionalDebug<true, "TCP/IP Stack error handler">;
+using DebugErrorHandler = ConditionalDebug<false, "TCP/IP Stack error handler">;
```

Comment:
8012640fa6732e3979fe448505b2e68e4264accc adds a `xmake.lua` `--debug-TCPIP` option that should probably be used here.